_redirects

```diff
@@ -99,7 +99,6 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 /upgrades-plugins/1.x/api-buidler-upgrades /upgrades-plugins/1.x/api-hardhat-upgrades 301
 
 # Defender V2 -> Netlify
-/defender/v2 https://defender-v2-docs-beta.netlify.app 301
-/defender/v2/* https://defender-v2-docs-beta.netlify.app/:splat 301
-# Redirection
-https://platform-docs-beta.netlify.app/platform/beta/ https://defender-v2-docs-beta.netlify.app/platform/beta/:splat 301
+/defender/v2 https://docs-v2--openzeppelin-defender.netlify.app 301
+/defender/v2/* https://docs-v2--openzeppelin-defender.netlify.app/defender/v2/:splat 301
+/defender/* /defender/v1/:splat 301
```

Comment:
This will be [generated automatically](https://github.com/OpenZeppelin/docs.openzeppelin.com/blob/master/scripts/latest-redirects.js) as long as v1 is the latest included version according to the playbook.

```suggestion
```